retrofit/build.gradle

```diff
@@ -26,7 +26,7 @@ jar {
 }
 
 // Create a test task for each supported JDK.
-(8..17).each { majorVersion ->
+(8..JavaVersion.current().majorVersion.toInteger()).each { majorVersion ->
```

Comment:
Doesn't this mean that if I'm running JDK 11 to build the project that I only get tests for 8-11?

Reply:
Yep

Reply:
The default test task running on the 18, and you added 8 ~ 17, so all tests running on 8~18? 
If so I will update this PR after JDK 19 release and change here to 18.

Reply:
Would like to wait for JDK 19 & Gradle 7.6 release, seems the Gradle team will support using Zulu in `jvmToolchain`.

Reply:
Can we set the toolchain explicitly to 18 rather than letting it be implicit? This would ensure that anyone building the project always builds on 18 and gets test tasks from 8-17.

Reply:
5d58a10f5a73a0e6f5149138e575668952471a5b